spek-runtime/src/main/kotlin/org/spekframework/runtime/scope/ScopeId.kt

```diff
@@ -0,0 +1,3 @@
+package org.spekframework.runtime.scope
+
+data class ScopeId(val type: String, val name: String)
```

Comment:
I've used an `enum` instead, since most of the time we will be using `class` and `scope`.